webapp/src/widgets/menu/textOption.tsx

```diff
@@ -18,6 +19,7 @@ function TextOption(props:TextOptionProps): JSX.Element {
     }
     return (
         <div
+            data-testid={props['data-testid']}
```

Comment:
I'm ok with this `data-testid` but probably we can use directly the `props.id` field there, instead of requiring an extra id.